source/crud/tests/v2/updateWithPipelines.yml

```diff
@@ -30,7 +38,7 @@ tests:
     outcome:
       collection:
         data:
-          - { _id: 1, x: 1, y: 1, u: {v: 1}, foo: 1 }
+          - { _id: 1, u: {v: 1}, foo: 1 }
```

Comment:
On second thought, the criteria for this `updateOne` may be ambiguous since it can match either of the two data fixture documents (both have `y: 1`). In practice, it's likely we'll match the first inserted document encountered by a scan but that's not technically guaranteed. I think it'd be safer to select by either `_id` or `x`. This has come up in previous CRUD tests and I recall we've made similar changes to avoid undefined behavior.

Reply:
I have changed this to select `_id` instead of `y`.